guides/v2.3/rest/modules/inventory/manage-source-selection.md

```diff
@@ -0,0 +1,140 @@
+---
+group: rest-api
+title: Manage source selection algorithms
+---
+
+Inventory Management uses the Source Selection Algorithm (SSA) to track the salable quantities of product inventory across all sources and make recommendations for partial and full shipments. 
+
+The SSA uses stocks and sources to check the sales channel for incoming product requests and determines the available inventory. It
+
+* Calculates the aggregated virtual salable quantity of all assigned sources per stock
+* Subtracts the Out-of-Stock Threshold amount from salable quantity to protect against overselling and support concurrent checkout
+* Reserves inventory quantities at checkout, deducting from in-stock inventory at order processing and shipment
+* Supports backorders with enhanced options for negative thresholds
+
+The SSA also manages shipments by providing recommendations for the best sources to ship partial or all products or override the selections to:
+
+* Ship partial shipments, sending only a few products from specific locations and completing the full order at a later date.
```

Comment:
Maybe change location to source? Just to keep it the same across all entries, etc?